lib/rules/number-max-precision/README.md

```diff
@@ -46,6 +46,30 @@ a { top: 3.24px; }
 
 ## Optional secondary options
 
+### `ignoreProperties: ["/regex/", /regex/, "string"]`
+
+Ignore the precision of numbers for the specified properties.
+
+Given:
+
+```
+["transition"]
+```
+
+The following patterns are considered violations:
+
+<!-- prettier-ignore -->
+```css
+a { -o-transition: all 4.5s ease; }
```

Comment:
```suggestion
a { top: 10.5px; }
```

Let's use a more common property as some users may not be familiar with the old Opera prefix.